cloud_functions/climateiq_trigger_export_pipeline_cf/main.py

```diff
@@ -27,44 +36,45 @@ def trigger_export_pipeline(cloud_event: http.CloudEvent) -> None:
     Args:
         cloud_event: The CloudEvent representing the storage event.
     """
+    start = time.time()
+
     data = cloud_event.data
     object_name = data["name"]
 
+    # Structured logging:
```

Comment:
nit: is this comment needed?

Reply:
Will clean up logging in a following PR.